js_modules/dagster-ui/packages/ui-core/src/instance/InstanceConcurrency.tsx

```diff
@@ -199,34 +293,31 @@ export const ConcurrencyLimits = ({
             <thead>
               <tr>
                 <th style={{width: '260px'}}>Concurrency key</th>
-                <th style={{width: '20%'}}>Occupied slots</th>
                 <th style={{width: '20%'}}>Total slots</th>
-                <th>Active runs</th>
+                <th style={{width: '20%'}}>Assigned steps</th>
+                <th style={{width: '20%'}}>Pending steps</th>
+                <th style={{width: '20%'}}>All steps</th>
                 <th></th>
               </tr>
             </thead>
             <tbody>
               {limits.map((limit) => (
                 <tr key={limit.concurrencyKey}>
                   <td>{limit.concurrencyKey}</td>
-                  <td>{limit.activeSlotCount}</td>
                   <td>{limit.slotCount}</td>
+                  <td>{limit.pendingSteps.filter((x) => !!x.assignedTimestamp).length}</td>
+                  <td>{limit.pendingSteps.filter((x) => !x.assignedTimestamp).length}</td>
                   <td>
-                    {limit.activeRunIds.length === 0 ? (
-                      <>&mdash;</>
-                    ) : (
-                      <Tag intent="primary" interactive>
-                        <ButtonLink
-                          onClick={() => {
-                            setSelectedKey(limit.concurrencyKey);
-                            setSelectedRuns(limit.activeRunIds);
-                          }}
-                        >
-                          {limit.activeRunIds.length}{' '}
-                          {limit.activeRunIds.length === 1 ? 'run' : 'runs'}
-                        </ButtonLink>
-                      </Tag>
-                    )}
+                    <span style={{marginRight: 16}}>{limit.pendingSteps.length}</span>
+                    <Tag intent="primary" interactive>
+                      <ButtonLink
+                        onClick={() => {
+                          setSelectedKey(limit.concurrencyKey);
+                        }}
+                      >
+                        View all
+                      </ButtonLink>
+                    </Tag>
```

Comment:
The tag wrapper on this seems a little unnecessary. It could just be a link, right? Also, I think you could combine the count and the link so that it's just a single `4 steps` button link.

I also noticed that the "zero" case in the table still has the link, but it seems like it probably doesn't need it.